### PR TITLE
multiple pages: ensure exactly one colon at the end of example descriptions

### PR DESCRIPTION
**Pull request by bl-ue**
Some example descriptions had no colons (or had periods) and some, but fewer, had two colons.

I didn't touch the Chinese or Japanese pages because I've learned that there is different punctuation for those languages (#5193) and I'll open a separate PR for them.

### CODE REVIEW

**Comment by bl-ue** [comment]
Funny. Alphabetically sorted, the first changed line has two spaces after the `-`.  I'll fix that in a new PR, but since it will be modifying a line that this PR has modified, that will cause conflicts, so I'll wait until this PR is merged.

File: pages.es/common/chmod.md
Comment (navarroaxel) [inline pages.es/common/chmod.md:5]:
```suggestion
- Otorga al [u]suario que es propietario del archivo permiso para [x] ejecutarlo:
```

File: pages.fr/common/apropos.md
Comment (navarroaxel) [inline pages.fr/common/apropos.md:13]: French needs an space:
```suggestion
- Recherche les pages qui contiennent toutes les expressions données (fonction ET) :
```

Comment (bl-ue) [reply pages.fr/common/apropos.md:13]: Oops, I forgot! You're right!  But is that the thin unbreakable space?

Comment (navarroaxel) [reply pages.fr/common/apropos.md:13]: oh nop.

Comment (bl-ue) [reply pages.fr/common/apropos.md:13]: It's this.  I'll edit and apply your suggestion.

Comment (navarroaxel) [reply pages.fr/common/apropos.md]: Obviously the French contributors don't use that char. We should add that to the linter for the French directory.

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/common/apropos.md
Comment (bl-ue) [reply pages.fr/common/apropos.md]: Some do, some don't.  You mean we're able to add non-English language-specific rules to the linter?

Comment (navarroaxel) [reply pages.fr/common/apropos.md]: sounds like a conditional rule based on the parent (of the parent) directory `pages.fr`. I think is possible to add that.

Comment (bl-ue) [reply pages.fr/common/apropos.md]: Hmm. That's really cool.  But this discussions should get it's own issue because if that's true, we really ought to extend some of the syntactic analysis that the linter can do to some of the other languages, because the punctuation of langs like Spanish/German are very similar to English...

Comment (bl-ue) [reply pages.fr/common/apropos.md]: I'm currently preparing a PR for normalizing the Chinese punctuation.  They have their own periods, commans, parentheses, semicolons, etc., and their own rules when mixing Latin characters and Chinese characters, etc., and it would be super great for that to be automated.  There's really no need for such tedious tasks to be done over and over again by a human ;)

File: pages.de/common/borg.md
Comment (sbrl) [inline pages.de/common/borg.md:15]: Double space
```suggestion
- Alle Archive in einem Repository auflisten:
```

**Review by sbrl — COMMENTED** [review]
Wow, awesome work @bl-ue!

Just found one teensy thing we can also fix, and then this is ready to go!

File: pages.de/common/borg.md
Comment (bl-ue) [reply pages.de/common/borg.md:15]: @sbrl I noticed that as well - please see my comment above https://github.com/tldr-pages/tldr/pull/5214#issuecomment-770426912

**Comment by schneiderl** [comment]
Given [this comment ](https://github.com/tldr-pages/tldr/pull/5214#issuecomment-770426912) I believe we can merge this PR.

**Review by schneiderl — APPROVED** [review]

**Pull request merged**
